docs/5.0/preview/database-access.md

```diff
@@ -149,34 +208,70 @@ ssh_service:
 Start the service:
 
 ```sh
-$ teleport start --debug --config=/path/to/teleport.yaml
+$ teleport start -d --config=/path/to/teleport.yaml
+```
+
+### Start Database Service with CLI Flags
+
+For a quick try-out, Teleport database service doesn't require a configuration
+file and can be launched using a single CLI command:
+
+```sh
+$ teleport start -d \
+   --roles=database \
+   --token=cbdeeab9-6f88-436d-a673-44d14bd86bb7 \
+   --auth-server=teleport.example.com:3080 \
+   --db-name=test \
```

Comment:
Does this need to match the name in the exported Teleport CA?

Reply:
No, this is just the service name used for identification purposes and in RBAC labels.